cmd/namespace/namespace.go

```diff
@@ -120,3 +126,16 @@ func askOktetoURL() (string, error) {
 
 	return oktetoURL, nil
 }
+
+func isNamespaceAvailable(ctx context.Context, namespace string) error {
+	client, _, err := k8Client.GetLocal()
+	if err != nil {
+		return err
+	}
+
+	_, err = namespaces.Get(ctx, namespace, client)
+	if err != nil {
+		return fmt.Errorf("Cannot get '%s' namespace. Please make sure it's created and you have the right permissions.", namespace)
```

Comment:
Could we add the second sentence as a `hint` instead?

```suggestion
		return fmt.Errorf("Namespace '%s' not found. Please verify that the namespace exists and that you have access to it.", namespace)
```